src/components/CurriculumComponents/CurriculumDownloadTab/index.tsx

```diff
@@ -1,5 +1,12 @@
 import { FC, useEffect, useLayoutEffect, useRef, useState } from "react";
-import { OakThemeProvider, oakDefaultTheme } from "@oaknational/oak-components";
+import {
+  OakDownloadsJourneyChildSubjectTierSelector,
+  OakThemeProvider,
+  oakDefaultTheme,
+  Tier,
+  Subject,
+} from "@oaknational/oak-components";
+import _ from "lodash";
```

Comment:
Better to pull though as `import {mapKeys, camelCase} from "lodash";` sometimes the bundler can do a bad job of working out what's actually included in the bundle otherwise.